sources.go

```diff
@@ -43,3 +44,14 @@ func Args() *Pipe {
 	}
 	return Echo(s.String())
 }
+
+// Get creates a pipe containing the output of retrieving the web resource
```

Comment:
Can you expand on this a bit (and also update the README)? See the documentation for existing methods for the sort of thing I'm looking for, including error handling, for example.